src/tools/mod.rs

```diff
@@ -7,8 +7,11 @@ pub mod synthetic_node;
 
 use std::time::Duration;
 
-/// Default timeout for connection reads in seconds.
+/// Default timeout for connection operations in seconds.
```

Comment:
the `TIMEOUT` name is a bit too broad IMO, same with `connection operations`; we should probably have one for starting a full node and, if we need another one for some connection-specific thing, have a separate one (with a more explicit name too) for it as well